### PR TITLE
Components

### PR DESCRIPTION
**Pull request by mmatongo**
#### This pull request aims to add/remove the following elements from the `dev-branch`

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new `Calculator.js` file.
- In Calculator.js, create a React component that matches this design:

<p align="center">
  Calculator
</p>

- Use class based components.
- Import `Calculator` class and use it as the main component (App.js).

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            <CalcButton text="8" />
+            <CalcButton text="9" />
+            <CalcButton text="4" />
+            <CalcButton text="5" />
+            <CalcButton text="6" />
+            <CalcButton text="1" />
+            <CalcButton text="2" />
+            <CalcButton text="3" />
+            <CalcButton text="0" />
+            <CalcButton text="." />
```
Comment (henatan99) [inline src/components/Calculator.js:31]: - [ ] These two buttons have slightly different styling than the original. Kindly compare and make changes to comply with the original design. :pray: Everything else is great!!! :clap: 

image

image

File: src/index.css [hunk 2]
```diff
@@ -1,27 +1,43 @@
...
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+.calc-screen {
+  text-align: right;
+  color: white;
+  background-color: #1c191c;
```
Comment (henatan99) [inline src/index.css:10]: - [ ] The background color for the `.calc-screen` class element is not as in the original design. Kindly compare and make changes accordingly. :+1: 

image
image

File: src/components/CalcButton.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+class CalcButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const { text } = this.props;
+    return (
+      <div className="calc-buttons">{text}</div>
```
Comment (henatan99) [inline src/components/CalcButton.js:13]: - [ ] I think it is semantically not right to use `div` tag for `button` elements. Kindly make changes here. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @mmatongo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**